app/code/Magento/Store/etc/di.xml

```diff
@@ -63,9 +63,11 @@
     </type>
     <preference for="Magento\Framework\App\ScopeResolverInterface" type="Magento\Store\Model\Resolver\Store" />
     <preference for="Magento\Framework\App\Router\PathConfigInterface" type="Magento\Store\Model\PathConfig" />
-    <type name="Magento\Framework\App\Action\AbstractAction">
-        <plugin name="storeCheck" type="Magento\Store\App\Action\Plugin\StoreCheck" sortOrder="10"/>
-        <plugin name="designLoader" type="Magento\Framework\App\Action\Plugin\Design" />
+    <type name="Magento\Framework\App\ActionInterface">
+        <plugin name="storeCheck" type="Magento\Store\App\Action\Plugin\StoreCheck"/>
+        <plugin name="designLoader" type="Magento\Framework\App\Action\Plugin\Design"/>
```

Comment:
Looks like it should be not in store module, but in di.xml related to framework - global di.xml file

Reply:
Actually - it should not be in `base` scope but `backend`.

Reply:
OK. There's different plugin for Backend. 
That one should be in Store and relate only to Frontend :man_facepalming:

Reply:
Managed to understand that and doing proper changes.